packages/dd-trace/test/appsec/attacker-fingerprinting.fastify.plugin.spec.js

```diff
@@ -26,13 +25,12 @@ withVersions('fastify', 'fastify', fastifyVersion => {
         reply.send('DONE')
       })
 
-      getPort().then((port) => {
-        app.listen({ port }, () => {
-          axios = Axios.create({ baseURL: `http://localhost:${port}` })
-          done()
-        })
-        server = app.server
+      app.listen({ port: 0 }, () => {
```

Comment:
Nit: Specifying no port and specifying `0` is the same in Node.js

```suggestion
      app.listen(() => {
```